code/modules/mob/living/silicon/ai/freelook/chunk.dm

```diff
@@ -98,6 +122,9 @@
 	src.y = y
 	src.z = z
 
+	for(var/turf_num in 1 to 16 * 16)
```

Comment:
The way you write this suggests there's some reason you chose specifically "16 * 16", when I was under the assumption you just pulled 256 because it's a nice number. Is there something I'm missing?

Reply:
each turf chunk covers 16x16 turfs although im now realizing its more clear if i just iterate the turfs list

Reply:
done